### ISSUE TITLE
cy.screenshot() or automatic screenshots on fail don't load images to RP when test name contains coma or single quote

### ISSUE THREAD

**Issue by chaoticsparks** [reportportal/agent-js-cypress#68]
To reproduce just add a spec file to this example with the following content:
```
context('screen', () => {
    it('test case title with ,', () => {
        cy.screenshot(); // or without that command
        expect(true).to.be.false;
    });

    it('test case title with \'', () => {
        cy.screenshot(); // or without that command
        expect(true).to.be.false;
    });
});
```

I tried to change the screenshot name in `after:screenshot` hook but it didn't work :(

**Comment by AmsterGet** [comment]
Hello @chaoticsparks !
The screenshot mechanism has been completely rewritten and released in version [5.1.2](https://github.com/reportportal/agent-js-cypress/releases/tag/v5.1.2).
Have you tried using it?
Please check your issue with the latest agent version.
Looking forward to your feedback!

**Comment by AmsterGet** [comment]
I'll close this issue.
Feel free to reopen it if your request is still relevant.
